pages/api-explorer/_meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "evm": "EVM",
+  "aptos": "Aptos",
+  "cosmwasm": "CosmWasm"
+}
```

Comment:
Suggest adding external links to the hermes and benchmarks docs endpoints as well

Reply:
done